portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,38 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <header>
+    <div id ="header">
+        <div id="center-alignment">
+            <h1> Sean Tisdale Portfolio </h1>
+        </div>
+        <div id="links">
+            <a href="projects.html"> Projects </a>
+        </div>
     </div>
+    </header>
+        <div id="content">
+           <div id="images">
+                <img src="images/self-image.JPG" alt="Picture of me ;)" width="300" height="400" style="border-radius:50%" style="padding: 20px" > 
+                <img src="images/selfie2.jpg" alt="Picture of me ;)" width="300" height="400" style="border-radius:50%" style="padding: 20px"> 
+           </div>
+           <br>
+            <p>Hello!! My name is Sean Tisdale, welcome to my personal webpage :D. I am a Computer Science major who atttends North Carolina A&T University and I am from Charlotte NC. I first got into Computer Science when I was in High School. During my junior
+                my school introduced a lot of new courses to take. Some of the ones I took were Computer Programming, Photoshop, and video game design. From these 3 courses I took over the school years 
+                computer programmming intereted me the most. After completing the course I did research and saw all of the things someone could do with computer science which always made me interested because sticking to one thing forever was always boring to me.
+                I am currently mostly interested in databases and AI in computer science. hobbies I have are playing games, watching anime or tv, and eating. I am currently trying to get into cooking more but it's a process so I'll give it time before calling it a hobby. 
+            </p>
+            <br>
+            <p>Click here to get a random message:</p>
+            <button onclick="addRandomFact()">Facts</button>
+            <div id="fact-container"></div>
+        </div>
```

Comment:
I think the footer is blocking this from being visible - http://screen/caztKrKwWD5.